pyeudiw/tests/settings.py

```diff
@@ -61,8 +61,8 @@
     },
     "trust": {
         "direct_trust_sd_jwt_vc": {
-            "module": "pyeudiw.trust.default.direct_trust_sd_jwt_vc",
-            "class": "DirectTrustSdJwtVc",
+            "module": "pyeudiw.trust.handler.direct_trust_sd_jwt_vc",
+            "class": "DirectTrustJWTHandler",
```

Comment:
DirectTrustJWTHandler doesn't give any good semantic within the context of `direct_trust_sd_jwt_vc`

using direct_trust_sd_jwt_vc the specialized class MUST DirectTrustSdJwtVc